qiskit/ignis/experiments/calibration/mock.py

```diff
@@ -13,62 +13,114 @@
+    duration: int = 160
+    amp: float = 0.04
+    phase: float = 0
+    sigma: float = 40
+    beta: float = 1.5
+    timestamp: pd.Timestamp = pd.Timestamp('2020-01-01 00:00:00')
+
+    def to_dict(self) -> Dict[str, Union[int, float]]:
+        """Return parameters as dictionary.
+
+        Returns:
+            Dictionary of parameters.
         """
-        name_map = {
-            'duration': default_duration,
-            'amp': default_amp,
-            'phase': default_phase,
-            'sigma': default_sigma,
-            'beta': default_beta
+        return {'duration': self.duration,
+                'amp': self.amp,
+                'phase': self.phase,
+                'sigma': self.sigma,
+                'beta': self.beta}
+
+
+@dataclass(frozen=True)
+class TwoQubitPulse:
+    channel: str = 'u0'
+    shape: str = 'gaussian_square'
+    duration: int = 800
+    amp: float = 0.05
+    phase: float = 0
+    sigma: float = 128
+    width: float = 544
+    timestamp: pd.Timestamp = pd.Timestamp('2020-01-01 00:00:00')
+
+    def to_dict(self) -> Dict[str, Union[int, float]]:
+        """Return parameters as dictionary.
+
+        Returns:
+            Dictionary of parameters.
+        """
+        return {'duration': self.duration,
+                'amp': self.amp,
+                'phase': self.phase,
+                'sigma': self.sigma,
+                'width': self.width}
+
+
+class FakeTwoQubitParameters(PulseTable):
+    """Mock pulse table database of two qubit system.
+
+    This system consists of x90p, x90m, y90p and cross resonance pulse from qubit0 to 1.
+    This configuration conforms to the typical IBM Quantum backend.
+    """
+    def __init__(self):
+        """Create new mock data base."""
+
+        instructions = {
+            'x90p': {
+                (0, ): SingleQubitPulse(channel='d0', amp=0.04),
+                (1, ): SingleQubitPulse(channel='d1', amp=0.05)
+            },
+            'x90m': {
+                (0, ): SingleQubitPulse(channel='d0', amp=0.04, phase=np.pi),
+                (1, ): SingleQubitPulse(channel='d1', amp=0.05, phase=np.pi)
+            },
+            'y90p': {
+                (0, ): SingleQubitPulse(channel='d0', amp=0.04, phase=np.pi / 2),
+                (1, ): SingleQubitPulse(channel='d1', amp=0.05, phase=np.pi / 2)
+            },
+            'xp': {
+                (0, ): SingleQubitPulse(channel='d0', amp=0.08),
+                (1, ): SingleQubitPulse(channel='d1', amp=0.10)
+            },
+            'cr90p_u': {
+                (0, 1): TwoQubitPulse(channel='u0', amp=0.05),
+            },
+            'cr90m_u': {
+                (0, 1): TwoQubitPulse(channel='u0', amp=0.05, phase=np.pi),
+            },
+            'cr90p_d': {
+                (0, 1): TwoQubitPulse(channel='d1', amp=0.02, phase=0.2)
+            },
+            'cr90m_d': {
+                (0, 1): TwoQubitPulse(channel='d1', amp=0.02, phase=0.2 + np.pi)
+            }
```

Comment:
Can you add the reverse instructions too? E.g. 
```
            'cr90m_u': {
                (1, 0): TwoQubitPulse(channel='u1', amp=0.05, phase=np.pi),
            },
```
Strictly speaking from the hardware perspective of a single backend it does make sense to do this. It does however makes it easier for testing with different backends as some have `(0,1)` and others have `(1,0)`.